README.md

```diff
@@ -13,85 +13,72 @@
-  <img src="https://user-images.githubusercontent.com/5100735/187891206-b32160f1-5f65-415e-869a-d314763d6f4b.svg" width="700"/>
+  <img src="https://user-images.githubusercontent.com/5100735/189058399-3865a039-9459-4e74-83dd-3ee2ecadfef5.svg" width="75%"/>
 </p>
 
 ## Why use `envd`?
 
-It is still too difficult to configure development environments and reproduce results in AI/ML applications.
-
-`envd` is a **machine learning development environment** for data science and AI/ML engineering teams. Environments built with `envd` provide the following features out-of-the-box:
+Environments built with `envd` provide the following features out-of-the-box:
 
-🐍 **Life is short, use Python[^1]/R/Julia**
+🐍 **Life is short, use Python[^1]**
 
-Development environments are full of Dockerfiles, bash scripts, Kubernetes YAML manifests, and many other clunky files that are always breaking. `envd` builds are isolated and clean. You can write simple instructions in Python/R/Julia, instead of Bash / Makefile / Dockerfile / ...
+Development environments are full of Dockerfiles, bash scripts, Kubernetes YAML manifests, and many other clunky files that are always breaking. `envd` builds are isolated and clean. You can develop with Python, save time on writing Bash / Makefile / Dockerfile / ...
 
-![envd](./docs/images/envd.png)
+![envd](https://user-images.githubusercontent.com/5100735/188821980-dcbd9069-b504-436a-9ffd-05ac5543a6d1.png)
 
 [^1]: The build language is [starlark](https://docs.bazel.build/versions/main/skylark/language.html), which is a dialect of Python.
 
-⏱️ **Save you plenty of time**
+⏱️ **6x faster build**
 
-`envd` adopts a multi-level cache mechanism to accelerate the building process. For example, the PyPI cache is shared across builds and thus the package will be cached if it has been downloaded before. It saves plenty of time, especially when you update the environment by trial and error.
+`envd` adopts a multi-level cache mechanism to accelerate the building process. For example, the PyPI cache is shared across builds and thus the package will be cached if it has been downloaded before. It saves plenty of time, especially when you update the environment by trial and error.[^2]
 
-<table>
-<tr>
-<td> envd </td> <td>
+<p align=center>
+  <img src="https://user-images.githubusercontent.com/5100735/188601795-8c37f5a3-b13b-422b-816f-8a0c51f1f8b1.svg" width="65%"/>
+</p>
 
-Docker[^2]
+[^2]: Docker without [buildkit](https://github.com/moby/buildkit)
 
-</td>
+☁️ **Local & cloud native**
+
+<table>
+<tr>
+<td>Local development simplifies the debugging, but...</td> 
+<td>Setup local & cloud native environment with envd</td>
 </tr>
 <tr>
 <td>
 
-```diff
-$ envd build
-=> pip install tensorflow       5s
-+ => Using cached tensorflow-...-.whl (511.7 MB)
-```
+❌ Can be complex to setup. When they break, you often need to run the whole setup.
 
-</td>
-<td>
+❌ Resource intensive because of the constraints to your CPU, moemory and local GPU.
```

Comment:
```suggestion
❌ Intensive resource because of your CPU, memory, and local GPU constraints.
```

Reply:
`Intensive resource` looks weird to me. I am not sure if it works.